src/mono/System.Private.CoreLib/src/System/TypeSpec.cs

```diff
@@ -312,18 +304,12 @@ private void AddName(string type_name)
             }
             else
             {
-                if (nested == null)
-                    nested = new List<ITypeIdentifier>();
+                nested ??= new List<ITypeIdentifier>();
                 nested.Add(ParsedTypeIdentifier(type_name));
             }
         }
 
-        private void AddModifier(IModifierSpec md)
-        {
-            if (modifier_spec == null)
-                modifier_spec = new List<IModifierSpec>();
-            modifier_spec.Add(md);
-        }
+        private void AddModifier(IModifierSpec md) => modifier_spec ??= new List<IModifierSpec>();
```

Comment:
@stephentoub was it intentional that you also removed the Add call?

Reply:
Definitely not.  Sorry.   I'll fix.

Reply:
https://github.com/dotnet/runtime/pull/72605